addons/website/static/src/components/dialog/duplicate_object.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import {useService, useAutofocus} from '@web/core/utils/hooks';
+
+import {WebsiteDialog} from './dialog';
+const {Component, useState, onWillStart} = owl;
+import {_t} from 'web.core';
+import wUtils from 'website.utils';
+
+export class DuplicateObjectDialog extends Component {
+    setup() {
+        this.orm = useService('orm');
+        this.website = useService('website');
+        this.notification = useService('notification');
+        useAutofocus();
+        this.state = useState({
+            name: '',
+            modalTitle: '',
+            objectName: '',
```

Comment:
It seems that `modalTitle` is not used, and I think `objectName` does not need to be in the state, as I don't see how it could change while the dialog is still displayed (maybe I am missing something, but in that case it could simply be a class property)